server/src/main/java/org/opensearch/transport/TransportService.java

```diff
@@ -161,6 +163,14 @@ public boolean isClosed() {
         public void close() {}
     };
 
+    static {
+        // registers server specific streamables
+        registerStreamables();
+    }
+
+    /** does nothing. easy way to ensure class is loaded */
+    public static void ensureClassloaded() {}
```

Comment:
Hm ... I am not sure why do we need that?

Reply:
It's for your comment below.